conf/base.config

```diff
@@ -59,4 +59,7 @@ process {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
+    withName: SEQKIT_STATS {
+        ext.args = ' ' // turn off --all default argument
```

Comment:
This is necessary because the module uses the `--all` argument by default, and we don't need/want the nucleotide-based columns such as `GC(%)`